build/bake/configuring-build.md

```diff
@@ -93,6 +93,57 @@ $ docker buildx bake -f docker-bake.hcl -f env.hcl --print app
 }
 ```
 
+You can also refer to attributes defined as part of other targets, to help
+reduce duplication between targets!
+
+```hcl
+# docker-bake.hcl
+target "foo" {
+  dockerfile = "${target.foo.name}.Dockerfile"
+  tags       = [target.foo.name]
+}
+target "bar" {
+  dockerfile = "${target.foo.name}.Dockerfile"
+  tags       = [target.bar.name]
+}
+```
+
+You can use this file directly:
+
+```console
+$ docker buildx bake --print foo bar
+```
```

Comment:
Add an empty line here

````suggestion
```

````

Reply:
We don't have a newline through the rest of the bake docs - I can add a newline after all of them, though it does make it clear that they're connected.

Reply:
Better to have a newline between block elements, just in case. Markdown parsers are inconsistent in interpreting this.

Reply:
Fixed :tada: